kotlin/services/rds/Readme.md

```diff
@@ -1,23 +1,119 @@
-# Amazon RDS Kotlin code examples
+# Amazon RDS code examples for the SDK for Kotlin
 
-This README discusses how to run the Kotlin code examples for Amazon Relational Database Service (Amazon RDS).
+This README discusses how to run and test the AWS SDK for Kotlin code examples for Amazon Relational Database Service (Amazon RDS).
 
-## Running the Amazon RDS Kotlin files
+Amazon RDS is a collection of managed services that makes it simple to set up, operate, and scale databases in the cloud.
 
-**IMPORTANT**
+## ⚠️ Important
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+*  We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
 
-The Kotlin code examples perform AWS operations for the account and AWS Region for which you've specified credentials, and you may incur AWS service charges by running them. See the [AWS Pricing page](https://aws.amazon.com/pricing/) for details about the charges you can expect for a given service and operation.
+## Code examples
 
-Some of these examples perform *destructive* operations on AWS resources, such as deleting a RDS instance. **Be very careful** when running an operation that deletes or modifies AWS resources in your account. It's best to create separate test-only resources when experimenting with these examples.
+The credential provider used in all code examples is Shared credentials. For more information, see [Credential providers](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/credential-providers.html).
 
-You will find these examples: 
+To run these examples, set up your development environment to use Gradle to configure and build AWS SDK for Kotlin projects. For more information, 
+see [Get started with the SDK for Kotlin](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/get-started.html).
```

Comment:
Does this belong under a Prerequisites header? 
[https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template)